src/Agent/NewRelic/Agent/Core/Transactions/NoOpTransaction.cs

```diff
@@ -81,6 +81,14 @@ public ISegment StartMessageBrokerSegment(MethodCall methodCall, MessageBrokerDe
             return Segment.NoOpSegment;
         }
 
+        public ISegment StartMessageBrokerSerializationSegment(MethodCall methodCall, MessageBrokerDestinationType destinationType, MessageBrokerAction operation, string brokerVendorName, string destinationName, string kind)
+        {
+#if DEBUG
+            Log.Finest("Skipping StartMessageBrokerSegment outside of a transaction");
```

Comment:
Do you think this will be useful to keep in?

Reply:
I think that keeping it in makes it consistent with the other Start*Segment methods. It may make sense to remove all of these logs in a separate story though. At finest level logs we should already be able to see that the current transaction is a noop transaction.